biosteam/_system.py

```diff
@@ -1624,54 +1628,70 @@ def clear_state(self):
             ws.state = y*0.0
             ws.dstate = y*0.0
 
-    def simulate(self, state_reset_hook=None,
-                 export_state_to='', sample_id='',
-                 print_msg=False,
-                 **kwargs):
+    def simulate(self, **dynsim_kwargs):
         """
         Converge the path and simulate all units.
         
         Parameters
         ----------
-        state_reset_hook: str or callable
-            Hook function to reset the cache state between simulations
-            for dynamic systems).
-            Can be "reset_cache" or "clear_state" to call `System.reset_cache`
-            or `System.clear_state`, or None to avoiding resetting.
-        export_state_to: str
-            If provided with a path, will save the simulated states over time to the given path,
-            supported extensions are '.xlsx', '.xls', 'csv', and 'tsv'.
-        sample_id : int or str
-            If `export_state_to` is provided with a valid path with extension
-            '.xlsx' or '.xls' , sample_id will be used as the sheet name.
-        kwargs : dict
-            Other keyword arguments that will be passed to ``solve_ivp``
-            or ``odeint``. Must contain t_span for ``solve_ivp`` or t for ``odeint``.
+        dynsim_kwargs : dict
+            Dynamic simulation keyword arguments, could include:
+
+                t_span : Iterable(float)
```

Comment:
might be better to just specify `2-tuple` in case an iterable of the wrong length is given

Reply:
It's kind of in the next line, but I'll change it to "tuple(float, float)"
> Interval of integration (t0, tf).